backend/src/test/resources/backend_config.json

```diff
@@ -75,14 +75,21 @@
                     {
                         "name": "booleanColumn",
                         "type": "boolean"
+                    },
+                    {
+                        "name": "insdcAccessionFull",
+                        "type": "string"
+                    },
+                    {
+                        "name": "other_db_accession",
```

Comment:
Let's keep casing consistent

Reply:
I don't disagree, but a) this predates me, was introduced by @anna-parker and b) is quite tangential to the PR as it's part of external metadata and the changes here are just to regularize test config with what we're doing in practice in Loculus/ppx

Reply:
Ah ok, I didn't see that it has to match with the external metadata below - but it's weird that one has to configure this field twice.

Reply:
Yeah as I say in the PR description, I think the backend config can be improved. I'm not a big fan of having external metadata being separate from metadata.

We should probably discuss in a separate issue with @anna-parker

It's easy to iterate over both metadata and external metadata here but the question is what's the semantics of the backend config.

Reply:
You can change to camelCase in both places if you would like